LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2022 Fabien Potencier
+Copyright (c) 2015-2023 Fabien Potencier
```

Comment:
See https://github.com/symfony/symfony/pull/49088
```suggestion
Copyright (c) 2015-present Fabien Potencier
```

Reply:
@stof sounds great! But what is the approach exactly? I see Symfony still uses  `2015-2023`, see https://github.com/symfony/symfony/blob/6.3/LICENSE#L1 - so I will keep the PR unchanged for now

Reply:
@bocharsky-bw see the Symfony PR linked at the beginning of that conversation. I know it is not merged yet, but it is already approved

Reply:
Now I see, thanks!